web/src/compose_closed_ui.js

```diff
@@ -183,10 +183,16 @@ export function initialize() {
 
     // Click handlers for buttons in the compose box.
     $("body").on("click", ".compose_new_conversation_button", () => {
-        compose_actions.start("stream", {trigger: "clear topic button"});
+        compose_actions.start({
+            trigger: "clear topic button",
+            message_type: "stream",
```

Comment:
Let's spell these with `message_type` as the first parameter; it's alphabetical and probably more natural logical ordering too.

Reply:
Sure :)